src/lib/helpers/renderConfig.test.ts

```
@@ -0,0 +1,181 @@
+
+export function isComponentConfig(value: any): value is RenderComponentConfig<any> {
+	return value instanceof RenderComponentConfig;
+}
+
+export function isSnippetConfig(value: any): value is RenderSnippetConfig<any> {
+	return value instanceof RenderSnippetConfig;
+}
+
+// Mock components for testing
+class MockComponent {
+	constructor(public props: any) {}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)